modules/core/arrow-data/src/main/kotlin/arrow/data/Ior.kt

```diff
@@ -28,7 +28,7 @@ typealias IorNel<A, B> = Ior<Nel<A>, B>
  * The isomorphic Either form can be accessed via the [unwrap] method.
  */
 @higherkind
-sealed class Ior<A, B> : IorOf<A, B> {
+sealed class Ior<out A, out B> : IorOf<A, B> {
```

Comment:
nice one

Reply:
It is like that on Master, I just fixed it back after breaking it xD